src/components/header/Header/Header.stories.tsx

```diff
@@ -0,0 +1,197 @@
+import React, { useState } from 'react'
+import { Header } from './Header'
+
+import { Title } from '../Title/Title'
+import { PrimaryNav } from '../PrimaryNav/PrimaryNav'
+import { Search } from '../../Search/Search'
+import { Menu } from '../Menu/Menu'
+import { MegaMenu } from '../MegaMenu/MegaMenu'
+import { NavButton } from '../NavButton/NavButton'
+import { NavDropDown } from '../NavDropDown/NavDropDown'
+
+export default {
+  title: 'Header/Header',
+  parameters: {
+    info: `
+USWDS 2.0 Header component
+
+Source: https://designsystem.digital.gov/components/header/
+`,
+  },
+}
+
+const mockSubmit = (): void => {
+  /* mock submit fn */
+}
+
+export const BasicHeader = (): React.ReactElement => {
+  const [expanded, setExpanded] = useState(false)
+  const onClick = (): void => setExpanded((prvExpanded) => !prvExpanded)
+
+  const testMenuItems = [
+    <a href="#linkOne" key="one">
+      <span>Current link</span>
+    </a>,
+    <a href="#linkTwo" key="two">
+      <span>Simple link Two</span>
+    </a>,
+  ]
+
+  const [isOpen, setIsOpen] = useState([false, false])
+
+  const onToggle = (index: number): void => {
+    setIsOpen((prevIsOpen) => {
+      const newIsOpen = Array(prevIsOpen.length).fill('false')
+      newIsOpen[index] = !prevIsOpen[index]
+      return newIsOpen
+    })
+  }
+
+  const testItemsMenu = [
+    <>
+      <NavDropDown
+        onToggle={(): void => {
+          onToggle(0)
+        }}
+        id="testDropDownOne"
+        isOpen={isOpen[0]}
+        label="Nav Label"
+      />
+      <Menu key="one" items={testMenuItems} isOpen={isOpen[0]} />
+    </>,
+    <a href="#two" key="two">
```

Comment:
Realized we need to use  buttons (not anchor tags) in our nav stories to ensure we match the nav item hover styles (include the underline on hover) with uswds. This is probably an issue in the footer too... so I'll go fix it there.

Reply:
FYI looked into this more in footer (there are also style discrepancies over there with `:hove`r) and it looks like the issue there was with the typography mixin not being applied in our repo.  I think this is probably a bigger issue - I'm either not understanding about how styles are working in our repo or something we haven't implemented yet.

Reply:
@haworku did you figure out what was going on here? lmk if you need another set of 👀

Reply:
@suzubara No I didn't -  more eyes would be helpful. Just comparing (for both header and footer) differences in hover styles in those stories versus behavior of the design system. 

 Header seems to be inheriting hover styles from css rules like this `usa-nav__primary > .usa-nav__primary-item > a:hover {
    color: #005ea2;`  

and this (which should apply if we use a button) `usa-nav__primary button[aria-expanded="false"]:hover` coming from `navigation.scss` . 

 Footer is inheriting hover styles like this `a:hover {
    color: #1a4480;
}` from `typography.scss`.

Reply:
Were you two able to get together and look into this issue?

Reply:
@eamahanna I compared your components with the markup at https://designsystem.digital.gov/components/header/#basic and using anchor tags here seems fine, but I think the links with spans inside them need to be switched ... So in other words, items inside of a dropdown should just be text inside of the `<a>`, without `<span>`, while links that are part of the top level nav _should_ have `<a><span>...`

Reply:
ah, yay! Good eyes.  I was jumbling up the dropdown and the simple links.